src/utils/TypeConversions.res

```diff
@@ -0,0 +1,22 @@
+let float_to_RString = (input: float) : React.element => {
+
```

Comment:
> {
  "[rescript]": {
    "editor.defaultFormatter": "chenglou92.rescript-vscode"
  },
  "files.insertFinalNewline": true,
  "editor.tabSize": 2,
  "editor.occurrencesHighlight": false,
  "editor.selectionHighlight": false,
  "reason_language_server.format_width": 100,
  "editor.formatOnSave": true,
  "editor.defaultFormatter": "jaredly.reason-vscode",
  "[scss]": {
    "editor.defaultFormatter": "esbenp.prettier-vscode"
  }
}

Use this as the `settings.json` file and save all the files in order to remove unnecessary spaces.